test/test_forms.py

```diff
@@ -431,17 +431,17 @@ def test_register_form_with_upref_mdata_disables_fields_correctly(self):
             'review'
         ]
         #Disable some fields
-        form.set_can_update((True, disabled))
+        form.set_can_update((True, disabled, {}))
```

Comment:
minor. passing none would work here matching other places or this test is trying to handle empty dict case?

Reply:
Agree, passing None would work and is consistent